pkg/producer/util.go

```diff
@@ -0,0 +1,194 @@
+// Copyright 2020 VMware, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package producer
+
+import (
+	"net"
+
+	"k8s.io/klog"
+
+	"github.com/vmware/go-ipfix/pkg/entities"
+	"github.com/vmware/go-ipfix/pkg/producer/protobuf"
+)
+
+type convertRecordToFlowType func(msg *entities.Message, record entities.Record) *protobuf.FlowMessage
```

Comment:
Do you think better to define a Go interface for converter, which includes a convert func?

Reply:
We can. Do you envision that more functions may be required in addition to converter function for each protobuf schema type?

Reply:
Just feel that is more OO design.

Reply:
Thanks for the suggestion. 
Sorry I could not get to this earlier. Addressed this comment. Could you take a look at this again?